Lombiq.Vsix.Orchard/Properties/AssemblyInfo.cs

```diff
@@ -15,22 +15,19 @@
 [assembly: AssemblyCopyright("")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-[assembly: ComVisible(false)]
+[assembly: ComVisible(true)]

```

Comment:
Kiderült, hogy ez kell VS extensionöknél?

Reply:
Túl sok információt nem találtam a témában, csak ezt a Stackoverflow kérdést: [https://stackoverflow.com/questions/46390727/in-a-visual-studio-extension-how-to-respond-to-open-folder-as-well-as-open-s](https://stackoverflow.com/questions/46390727/in-a-visual-studio-extension-how-to-respond-to-open-folder-as-well-as-open-s)
Viszont úgy tűnik, hogy a ComVisiblity(false) nélkül is működik.